pkg/platform/aws/aws.go

```diff
@@ -106,6 +106,31 @@ func NewConfig() *config {
 	}
 }
 
+func (c *config) configureKubeAPIServerExtraFlags() error {
```

Comment:
This seems like a lot of boilerplate code, which needs to be almost the same for all platforms. We should refactor that somehow.